File: Player.py
class Player:
    ## Creating player object to store name and current position of player
    def __init__(self,name):
        self.name=name
        self.position=0

    def getName(self):
        return self.name
    
    def setPosition(self,pos):
        self.position=pos
        
    def getPosition(self):
        return self.position

File: Player.java
public class Player {
    private String name;
    private int position;

    Player(String name){
        this.name=name;
        this.position=0;
    }
    
    public String getName() {
        return name;
    }

    public int getPosition() {
        return position;
    }
    public void setPosition(int position) {
        this.position = position;
    }
}
